includes/Modules/Analytics.php

```diff
@@ -126,6 +126,80 @@ function () {
 			},
 			0
 		);
+
+		add_action(
+			'admin_init',
+			function() {
+				$this->handle_provisioning_callback();
+			}
+		);
+	}
+
+	/**
+	 * Handles the provisioning callback after the user completes the terms of service.
+	 *
+	 * @since 1.0.0
+	 */
+	private function handle_provisioning_callback() {
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			return;
+		}
+
+		$input = $this->context->input();
+
+		if ( $input->filter( INPUT_GET, 'gatoscallback' ) ) {
+			// The handler should check the received Account Ticket id parameter against the id stored in the provisioning step.
+			$account_ticket_id        = $this->context->input()->filter( INPUT_GET, 'accountTicketId', FILTER_SANITIZE_STRING );
+			$stored_account_ticket_id = get_transient( PROVISION_ACCOUNT_TICKET_ID . '::' . get_current_user_id() );
```

Comment:
This needs to be the class constant, this constant is no longer in global namespace per #1365

Reply:
Right, changing.